### PR TITLE
75 - Fix syntax issue and update instructions in quickstart for JS

### PR DESCRIPTION
**Pull request by abubakarsohail**
This pull request fixes a syntax issue and updates the instructions in the quickstart guide for JavaScript. The syntax issue was causing an error in the code, and the updated instructions provide clearer guidance for setting up the project.

### CODE REVIEW

File: quickstart.mdx [hunk 1]
```diff
@@ -51,13 +51,15 @@ openai.Completion.create(
 // Make sure to `npm install promptlayer`
 import { promptlayer } from "promptlayer";
 
+// This assumes you already have `openai` installed
+// If not, you can install it with `npm install openai`
```
Comment (jzone3) [inline quickstart.mdx:55]: Replace with:

// Make sure you have openai installed with `npm install openai`

File: quickstart.mdx [hunk 2]
```diff
@@ -51,13 +51,15 @@ openai.Completion.create(
...
 // Swap out your 'import openai'
 const OpenAI = promptlayer.OpenAI;
 const openai = new OpenAI();
 
 // Do something fun 🚀
 openai.completions.create({
-  model: "gpt-3.5-turbo-instruct"
+  model: "gpt-3.5-turbo-instruct",
   prompt: "My name is",
 })
```
Comment (jzone3) [inline quickstart.mdx:64]: Let's add the then and catch stuff Ian had?

**Review by jzone3 — COMMENTED** [review]
Quick comments

can you also add:
"Grab an API key from OpenAI."
under the promptlayer bullet on the quickstart page?

File: quickstart.mdx (hunk 1 above)
Comment (abubakarsohail) [reply quickstart.mdx:55]: Done

**Comment by abubakarsohail** [comment]
> Quick comments
> 
> can you also add: "Grab an API key from OpenAI." under the promptlayer bullet on the quickstart page?
Updated

**Review by jzone3 — APPROVED** [review]

**Pull request merged**